frontend/src/pages/Auth/AdminForm.tsx

```diff
@@ -187,91 +188,74 @@ export const AdminForm: React.FC = () => {
 		<Landing>
 			<Form
 				className={authRouterStyles.container}
-				state={formState}
+				store={formStore}
 				resetOnSubmit={false}
 			>
 				<AuthHeader>
 					<Text color="moderate">Tell us a bit more</Text>
 				</AuthHeader>
 				<AuthBody>
 					<Stack gap="12">
-						<Form.NamedSection
-							label="Your Name"
-							name={formState.names.role}
-						>
```

Comment:
Noticed this was incorrectly referencing `role` instead of `firstName`, also doesn't make sense to wrap multiple inputs with a label.